source/partials/release_notes/_release-16-9-0.html.erb

```diff
@@ -0,0 +1,139 @@
+<h4>Quick Edit (Beta)</h4>
+
+<p>Currently, editing a pipeline means navigating through multiple pages (and page refreshes), for modifying stages, jobs with tabs for options and environment variables, etc. Quick edit to make it smoother.</p>
+
+<p>With this feature, you can change a pipeline’s materials/stages/jobs/tasks simultaneously and update it as single operation. This feature also uses the new pipeline config API to make your configuration changes quicker.</p>
+
+<p>With this feature, you can change a pipeline’s materials/stages/jobs/tasks simultaneously and update it as single operation. This feature also uses the new pipeline config API to make your configuration changes quicker.</p>
+
+<b>How do I use this?</b>
+
+<p>Click on "Quick Edit" button on the pipeline edit page.</p>
+
+<figure>
+  <%= image_tag "releases/16.9/quickeditbutton.png", {:title => "Quick edit", :alt => "Quick edit"} %>
+</figure>
+
+<p>Click on "Normal Edit" button to get back to old edit page.</p>
+<figure>
+  <%= image_tag "releases/16.9/quickeditpage.png", {:title => "Quick edit", :alt => "Quick edit"} %>
+</figure>
+
+<b>Why is this beta?</b>
+
+<p>We are working on adding few missing features like:</p>
+
+<ul>
+	<li>Package materials (like yum, maven, artifactory plugins)</li>
+	<li>Pipeline template editing, pipelines that use templates can only update Pipeline Settings 
+and Materials</li>
+</ul>
+
+<b>Known issues?</b>
+
+<p>Missing API validation for a few attributes, they are tracked below:</p>
+
+<ul>
+	<li>Pipeline Config API: Stage operate permission user not validated (<%= link_to_issue('2629') %>)</li>
+	<li>Pipeline Config API: Validation missing for parameter name used in the custom task argument (<%= link_to_issue('2628') %>)</li>
+	<li>Missing validation errors on pipeline_config API (<%= link_to_issue('2553') %>)</li>
+</ul>
+
+<b>What’s next?</b>
+
+<p>We really appreciate your feedback to make this page better.</p>
+
+<p>Going forward, once all the features are in, we plan to use this page as the default view for modifying configuration.</p>
+
+<p>Submit your feedback by creating github issue at https://github.com/gocd/gocd/issues</p>
+
+<h4>API Enhancements</h4>
+<p><%= link_to_commit('d662685', 'Provided Git and git-gc stats in the support api ') %>(<%= link_to_issue('2578') %>)</p>
+
+<h4>Library upgrades</h4>
+<p><%= link_to_commit('973cbf2', 'Upgraded jetty minor version to 9.2.18.v20160721')%></p>
+<p><%= link_to_commit('74b8b47', 'Upgraded logging jars')%></p>
+
+<h4>Bug fixes</h4>
+<p><%= link_to_commit('dc78601', 'Added validation of the encryptedPassword attribute of password aware materials - svn, p4, tfs.') %></p>
+
+<h4>Security Fixes</h4>
+<p><%= link_to_commit('faa6a2a', 'Removed displaying url’s from the not found page ') %>(<%= link_to_issue('2584') %>)</p>
```

Comment:
added the space so that it looks like
 
"from the not found page (2584)"

Removing it makes it

"from the not found page(2584)"